graphene_sqlalchemy/converter.py

```diff
@@ -147,21 +147,17 @@ def convert_column_to_float(type, column, registry=None):
 
 @convert_sqlalchemy_type.register(types.Enum)
 def convert_enum_to_enum(type, column, registry=None):
-    enum_class = getattr(type, 'enum_class', None)
-    if enum_class:  # Check if an enum.Enum type is used
-        graphene_type = Enum.from_enum(enum_class)
-    else:  # Nope, just a list of string options
-        items = zip(type.enums, type.enums)
-        graphene_type = Enum(type.name, items)
-    return Field(
-        graphene_type,
-        description=get_column_doc(column),
-        required=not (is_column_nullable(column)),
-    )
+    if registry is None:
+        from .registry import get_global_registry
```

Comment:
I think this has been done to avoid a circular import. Here in the converter module it works when importing on top (I will change that) but in the utility module it needs to be done that way, because registry imports utility.